rust/noosphere-gateway/src/worker/syndication.rs

```diff
@@ -89,15 +91,19 @@ where
         };
         let checkpoint_key = format!("syndication/kubo/{kubo_identity}");
 
+        debug!("IPFS node identified as {}", kubo_identity);
```

Comment:
We should set some RUST_LOG level for gateways in the cluster to verify and get insight in deployment

Reply:
SG. By default it is set for `orb`, but the scope is somewhat limited in scope to a few crates.